packages/v4/src/content/community/extensions-data.js

```diff
@@ -0,0 +1,58 @@
+export const communityExtensions = [
+  {
+    "component": "Catalog View",
+    "description": "Implements a catalog view using PatternFly components",
+    "tags": ["App services"],
+    "links": [
+      {
+        "name": "Doc",
+        "href": "https://patternfly-react.surge.sh/extensions/catalog-view-item-header"
+      },
+      {
+        "name": "Code",
+        "href": "https://www.npmjs.com/package/@patternfly/react-catalog-view-extension"
+      }
+    ]
+  },
+  {
+    "component": "Console",
+    "description": "React-console extension",
+    "tags": ["App services"],
+    "links": [
+      {
+        "name": "Doc",
+        "href": "https://patternfly-react.surge.sh/extensions/access-consoles"
+      },
+      {
+        "name": "Code",
+        "href": "https://github.com/patternfly/patternfly-react/tree/master/packages/react-console"
+      }
+    ]
+  },
+  {
+    "component": "Topology View",
+    "description": "Implements a topology view",
```

Comment:
```suggestion
    "description": "Implements a topology view.",
```

Let's punctuate this description since it forms a full sentence, similar to our descriptions under component callout images in our design guidelines.